segment.py

```diff
@@ -48,17 +50,19 @@ def lambda_handler(event, context):
             # )
 
             # Call ffmpy function
-            segment()
+            segment(LocalPath)
             global destination
 
             # Each chunk is uploaded to s3
+            FilePath, Extension = os.path.splitext(LocalPath)
             print "Uploading segments to s3..."
-            for filename in os.listdir('/tmp/SEGMENT*'):
-                destination = 'Segmented/{}'.format(filename)
-                s3_client.upload_file('/tmp/'+filename, bucket, destination)
+            for filename in os.listdir('/tmp/{}/{}*'.format(ConversionID, FilePath)):
+                if !filename.endswith('.mp3'):
+                    destination = 'Segmented/{}'.format(filename)
+                    s3_client.upload_file('/tmp/'+filename, bucket, destination)
             print "Uploading audio to s3"
             audiodestination = 'Audio/{}'.format(file_name)
-            s3_client.upload_file('/tmp/'+file_name+'.mp3', bucket, audiodestination)
+            s3_client.upload_file('{}.mp3'.format(FilePath), bucket, audiodestination)
```

Comment:
We should upload to the same path we found the file originally, but in a folder. - A/B/C/D/ABCDOUEBOAWEUB/segment1.mp4